services/ui/streamlit_app.py

```
@@ -1635,15 +1682,48 @@ def _delete_site_and_clear() -> None:
+        from app import job_store
+
+        ws = Path(job_store.workspace_root())
+        jobs_root = ws / "jobs"
+        if jobs_root.is_dir():
+            for meta_path in jobs_root.glob("*/job.json"):
+                try:
+                    import json
+
+                    meta = json.loads(meta_path.read_text(encoding="utf-8"))
+                except Exception:
+                    continue
```

Ruff (0.16.0)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)